src/hina.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<meta name="description" content="Update your description here" />
+	<title>Pavementcoffeehouse</title>
+	<!--
+		Webpack will automatically link your CSS file for you.
+		Work in the src/css/styles.scss file.
+	-->
+	<style>
+		@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap');
+		a {
+			text-decoration: none;
+		}
+	</style>
+</head>
+
+<body>
+
+		<div class="navigation">
+			<img src="images/logo.jpg" alt="pavement logo">
+		 <nav>
+			<a href="#order" class="nav">Order</a>
+			<a href="#About us" class="nav">About us</a>
+			<a href="#Location" class="nav">Location</a>
+		 </nav>
+		 <div class="cart">
+			<img src="images/shopping-cart.jpg" alt="shopping cart">
+		</div>
+		  
+		</div>
+			<div class="heading">
+			<img src="images/cup.jpg" alt="image of cup">
+				<h1>Would you like to add any flavors to your coffee?</h1>
+			</div>
+		  
+			<section class="coffee flavor options">
+				<img src="images/hazelnut.jpg" alt="image of Hazelnut">
+				<p>Hazelnut</p>
```

Comment:
Think about the selections on this page. Should the user only be allowed to select one, or can they select multiple? You'll need a different type of HTML element - either a radio or a checkbox - depending on the answer. Swap that out for the p tag as soon as you're able to. https://www.w3schools.com/howto/howto_css_custom_checkbox.asp